aws-iam-role-ec2-poweruser/variables.tf

```diff
@@ -25,6 +25,18 @@ variable "saml_idp_arn" {
   description = "The AWS SAML IDP arn to establish a trust relationship. Ignored if empty or not provided."
 }
 
+variable "iam_actions" {
+  type        = list(string)
+  default     = ["ec2:*"]
+  description = "List of EC2 API Actions that we are granting this role. "
```

Comment:
This could be somewhat "abused" to grant perms outside the ec2 scope. I wonder if we should consider forcing the ec2 namespace by doing something like `formatlist("ec2:%s", var.iam_actions)` and then the default action is `["*"]`

Reply:
Ah yeah, that's a good point. I'll add this